tendrl-node-agent.spec

```diff
@@ -23,6 +23,8 @@ Requires: python-greenlet
 Requires: collectd
 Requires: python-jinja2
 Requires: tendrl-common
+Requires: hwinfo
+Requires: python-netifaces
```

Comment:
hwinfo not available, neti faces is available, hwinfo we have to build, @brainfunked suggested hwinfo to collect hardware inventory details